### PR TITLE
fix: replace bootstrap variables with neptune tokens

### PR DESCRIPTION
**Pull request by anttio**
☝ This title will appear in the changelog - keep it meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint).

## 🖼 Context
We are deprecating Bootstrap so there's a need to start using Neptune tokens and variables.

## 🚀 Changes
Replaced all imported Bootstrap variables with Neptune Tokens and LESS variables.

## ✅ Checklist

- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/master/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

File: packages/components/src/navigationOption/NavigationOption.less [hunk 1]
```diff
@@ -1,4 +1,5 @@
-@import 'bootstrap/less/variables';
+@import '@transferwise/neptune-css/src/less/variables/_colors.less';
+@import '@transferwise/neptune-css/src/less/variables/_typography.less';
```
Comment (helendurranttw) [inline packages/components/src/navigationOption/NavigationOption.less:2]: Looks like maybe this isn't needed any more?

File: packages/components/src/phoneNumberInput/PhoneNumberInput.less [hunk 2]
```diff
@@ -1,16 +1,18 @@
-@import 'bootstrap/less/variables';
+@import '@transferwise/neptune-css/src/less/variables/_custom-media.less';
```
Comment (helendurranttw) [inline packages/components/src/phoneNumberInput/PhoneNumberInput.less:1]: Think you can remove this now too

File: packages/components/src/stepper/Stepper.less [hunk 3]
```diff
@@ -1,33 +1,35 @@
-@import 'bootstrap/less/variables';
+@import '@transferwise/neptune-tokens/colors-base.less';
+@import '@transferwise/neptune-css/src/less/variables/_custom-media.less';
```
Comment (helendurranttw) [inline packages/components/src/stepper/Stepper.less:2]: I think these two can go

File: packages/components/src/tabs/Tabs.less [hunk 4]
```diff
@@ -1,5 +1,5 @@
-@import 'bootstrap/less/variables.less';
 @import '@transferwise/neptune-css/src/less/variables/_colors.less';
+@import '@transferwise/neptune-css/src/less/variables/_custom-media.less';
```
Comment (helendurranttw) [inline packages/components/src/tabs/Tabs.less:2]: And this?

File: packages/components/src/navigationOption/NavigationOption.less (hunk 1 above)
Comment (anttio) [reply packages/components/src/navigationOption/NavigationOption.less:2]: There's the link hover and active state colours that are not yet part of the tokens package, thus the `_colors.less` is needed.

`_typography.less` is needed for font-weight change in the media-body.

These imports can be removed when we have them in the tokens package.

File: packages/components/src/phoneNumberInput/PhoneNumberInput.less (hunk 2 above)
Comment (anttio) [reply packages/components/src/phoneNumberInput/PhoneNumberInput.less:1]: I think this is still needed to get the media queries to work.

File: packages/components/src/stepper/Stepper.less (hunk 3 above)
Comment (anttio) [reply packages/components/src/stepper/Stepper.less:2]: Still needed until everything is in tokens.

File: packages/components/src/tabs/Tabs.less (hunk 4 above)
Comment (anttio) [reply packages/components/src/tabs/Tabs.less:2]: I think this is still needed to get the media queries to work.

File: packages/components/src/dynamicFieldDefinitionList/FormattedValue/FormattedValue.less [hunk 5]
```diff
@@ -1,5 +1,3 @@
-@import 'bootstrap/less/variables';
-
 .formatted-value__h3-custom-alignment {
   margin-bottom: 16px;
   line-height: 24px;
```
Comment (dozortsev) [inline packages/components/src/dynamicFieldDefinitionList/FormattedValue/FormattedValue.less:3]: shouldn't those two be using variables?

File: packages/components/src/flowNavigation/backButton/BackButton.less [hunk 6]
```diff
@@ -1,4 +1,4 @@
-@import 'bootstrap/less/variables';
+@import '@transferwise/neptune-tokens/colors-base.less';
 
 @transition-timing-function: ease-in-out;
 @transition-duration: 0.3s;
```
Comment (dozortsev) [inline packages/components/src/flowNavigation/backButton/BackButton.less:4]: what about this one? we have `--transition-duration: 350ms;` in `_transitions.less` file

File: packages/components/src/summary/Summary.less [hunk 7]
```diff
@@ -1,5 +1,3 @@
-@import '@transferwise/neptune-css/src/less/variables/_colors.less';
-@import '@transferwise/neptune-css/src/less/variables/_spacing.less';
 @import '@transferwise/neptune-css/src/less/variables/_typography.less';
```
Comment (dozortsev) [inline packages/components/src/summary/Summary.less:1]: why we removing those two and leaving `_typography.less` ?

File: packages/components/src/typeahead/Typeahead.less [hunk 8]
```diff
@@ -1,14 +1,19 @@
-@import 'bootstrap/less/variables';
+@import '@transferwise/neptune-css/src/less/variables/_custom-media.less';
+@import '@transferwise/neptune-css/src/less/variables/_sizes.less';
+@import '@transferwise/neptune-css/src/less/variables/_zindex.less';
+
 @animation-timing-function: ease-in-out;
 @animation-duration: 0.03s;
```
Comment (dozortsev) [inline packages/components/src/typeahead/Typeahead.less:6]: I guess those two can be delete and instead use variables from `_transitions.less`

File: packages/components/src/dynamicFieldDefinitionList/FormattedValue/FormattedValue.less (hunk 5 above)
Comment (anttio) [reply packages/components/src/dynamicFieldDefinitionList/FormattedValue/FormattedValue.less:3]: Margin-bottom replaced with space-16. For line-heights we don't have fixed sized variables.

File: packages/components/src/flowNavigation/backButton/BackButton.less (hunk 6 above)
Comment (anttio) [reply packages/components/src/flowNavigation/backButton/BackButton.less:4]: We can revisit these LESS variables later. The variables in `_transitions.less` are almost the same, but might affect the animations.

File: packages/components/src/typeahead/Typeahead.less (hunk 8 above)
Comment (anttio) [reply packages/components/src/typeahead/Typeahead.less:6]: Let's revisit these LESS variables later.

File: packages/components/src/select/Select.less [hunk 9]
```diff
@@ -10,27 +11,27 @@ html.ios-click {
...
-        margin-right: @spacer;
+        margin-right: var(--space-8);
       }
     }
 
     &--divider {
       border-bottom-style: solid;
       border-bottom-width: 1px;
-      border-bottom-color: @brand-smoke-plus-20;
+      border-bottom-color: @color-base-smoke-fade;
```
Comment (andreapiras) [inline packages/components/src/select/Select.less:21]: Can we not use `--color-background-neutral` here from @import '@transferwise/neptune-tokens/colors.css'; and import @transferwise/neptune-tokens/colors.css with the custom plugin u added?

File: postcss.config.js [hunk 10]
```diff
@@ -1,14 +1,20 @@
...
+require('@arshaw/postcss-custom-properties');
 require('postcss-custom-media');
 require('cssnano');
 
 module.exports = {
   plugins: {
     'postcss-import': {},
     autoprefixer: {},
-    'postcss-custom-properties': {},
+    '@arshaw/postcss-custom-properties': {
```
Comment (andreapiras) [inline postcss.config.js:11]: 😍

File: packages/components/src/select/Select.less (hunk 9 above)
Comment (anttio) [reply packages/components/src/select/Select.less:21]: We should use the `--color-border-*` semantic colours here, but we don't have that lighter colour there yet.

**Pull request merged**